MaxText/configs/base.yml

```diff
@@ -261,3 +261,8 @@ vertex_tensorboard_project: ""
 # Region to create Vertex AI Tensorboard in for GCE, blank if running via XPK
 # Vertex AI supported regions: https://cloud.google.com/vertex-ai/docs/general/locations#available-regions
 vertex_tensorboard_region: ""
+
```

Comment:
Not in this PR, but I see a need for separate inference specific config files in future -- both base.yml and model specific config.